package.json

```diff
@@ -16,6 +16,7 @@
   "license": "ISC",
   "dependencies": {
     "express": "^4.18.2",
+    "normalize.css": "^8.0.1",
```

Comment:
What is this dependency? Is it being used somewhere?

Reply:
ah, normalize.css is a sorta css reset I installed the other night, turned it off for this.